lib/LANraragi/Model/Tankoubon.pm

```diff
@@ -509,4 +493,41 @@ sub get_tankoubons_containing_archive ($arcid) {
     return @tankoubons;
 }
 
+sub tankoubon_tools ( $action, $tank_id, $field, $value ) {
```

Comment:
Wouldn't this be better as just two distinct subs?

Reply:
Ok, I joined as a way to have a single method to do all operations related to metadata, but it can be done separated.